spec.html

```diff
@@ -43183,13 +43157,13 @@ <h1>VariableStatements in Catch Blocks</h1>
         <p>The |Block| of a |Catch| clause may contain `var` declarations that bind a name that is also bound by the |CatchParameter|. At runtime, such bindings are instantiated in the VariableDeclarationEnvironment. They do not shadow the same-named bindings introduced by the |CatchParameter| and hence the |Initializer| for such `var` declarations will assign to the corresponding catch parameter rather than the `var` binding.</p>
       </emu-note>
       <p>This modified behaviour also applies to `var` and `function` declarations introduced by direct eval calls contained within the |Block| of a |Catch| clause. This change is accomplished by modifying the algorithm of <emu-xref href="#sec-evaldeclarationinstantiation"></emu-xref> as follows:</p>
-      <p>Step 5.d.ii.2.a.i is replaced by:</p>
+      <p>Step 3.d.i.2.a.i is replaced by:</p>
       <emu-alg type="i">
-        1. If _thisEnvRec_ is not the Environment Record for a |Catch| clause, throw a *SyntaxError* exception.
+        1. If _thisLex_ is not the Environment Record for a |Catch| clause, throw a *SyntaxError* exception.
       </emu-alg>
-      <p>Step 9.d.ii.4.b.i.i is replaced by:</p>
+      <p>Step 7.d.ii.4.a.i.i is replaced by:</p>
```

Comment:
I don't know what this is supposed to be referring to. EvalDeclarationInstantiation has a step 7 that notes alternative semantics from Annex B, but not from B.3.5. None of the other steps even look like they could have been named this. I'm super confused here.

Reply:
Yeah, that's a tricky one. [EvalDeclarationInstantiation](https://tc39.es/ecma262/#sec-evaldeclarationinstantiation)'s step 9 refers to [B.3.3.3](https://tc39.es/ecma262/#sec-web-compat-evaldeclarationinstantiation), so the latter's step 1.d.ii.4.b.i.i becomes the former's 9.d.ii.4.b.i.i, which [B.3.5](https://tc39.es/ecma262/#sec-variablestatements-in-catch-blocks) then modifies.